blog/2023-08-10-evaluating-open-source-llms/index.md

```diff
@@ -0,0 +1,148 @@
+
+On the other hand, if privacy and security are absolutely required, then you can host your own open-source LLM. Along with using the benchmarks, you'll have to experiment with a handful of LLMs yourself to get a better idea of how each performs on live data. 
+
+
+## Reinforcement Learning
+
+Open-source models don't have the same size as ChatGPT or GPT-4, but even the larger 70B parameter models can have issues related to bias and fairness. When GPT-3 was first released it has [racial biases, gender biases](https://aclanthology.org/2021.nuse-1.5/), and [religious biases](https://hai.stanford.edu/news/rooting-out-anti-muslim-bias-popular-language-model-gpt-3). The origin of these biases was the data these models were trained on.
+
+It's nearly impossible to remove all biases from a dataset, but at the same time, we don't want the models to pick up on them. How do we fix this problem?
+
+This is where [*Reinforcement Learning with Human Feedback(RLHF)*](https://wandb.ai/ayush-thakur/RLHF/reports/Understanding-Reinforcement-Learning-from-Human-Feedback-RLHF-Part-1--VmlldzoyODk5MTIx) comes in. With RLHF, a human ranks the outputs of an LLM from best to worst. Each time the human ranks the outputs, they are essentially training a reinforcement model. This reinforcement model is then used to "reward" the model when it generates an output that is "good" and penalizes the model when the output is "bad".
+
+Using RLHF, the hidden biases a model has picked up from the training dataset can be removed, hence improving the model's accuracy.
+
+Pros:
+- Increased efficiency: Using RLHF, the feedback that is provided helps guide the LLM toward the optimal solution. With only a few examples, a fine-tuned model with RLHF can easily outperform the baseline model on certain tasks.
+- Better Performance: The feedback that a human provides also impacts the quality of the output generated by an LLM. By showing more examples of the desired outcomes, the LLM improves the generated output to match what is expected.
+
+Cons:
+- Lack of scalability: RLHF depends on human feedback to improve the performance of the model. So, in this case, the human is the bottleneck. Providing feedback to an LLM can be a time-consuming process and it can't be automated. Because of this, RLHF is considered a slow and tedious process.
+- Inconsistent quality: Different people may be providing feedback for the same model and those people may have differing opinions on what should be the desired output. People make decisions based on their knowledge and preference, but too many differing opinions can confuse the model and lead to performance degradation.
```

Comment:
would love a citation for "too many differing opinions can confuse the model and lead to performance degradation"